interface/patient_file/summary/demographics_full.php

```diff
@@ -1,1058 +1,1055 @@
-
- var i = string.indexOf(text);
- if ((!i) && (text != string.substring(0,txtLength))) return string;
- if (i == -1) return string;
-
- var newstr = string.substring(0,i) + by;
-
- if (i+txtLength < strLength)
-  newstr += replace(string.substring(i+txtLength,strLength),text,by);
-
- return newstr;
-}
-
-function upperFirst(string,text) {
- return replace(string,text,text.charAt(0).toUpperCase() + text.substring(1,text.length));
-}
-
-<?php 
-$count = getActiveInsuranceData();
-for ($i=0;$i<count($count);$i++) { ?>
-function auto_populate_employer_address<?php echo $i ?>(){
- var f = document.demographics_form;
- if (f.form_i<?php echo $i?>subscriber_relationship.options[f.form_i<?php echo $i?>subscriber_relationship.selectedIndex].value == "self")
- {
-  f.i<?php echo $i?>subscriber_fname.value=f.form_fname.value;
-  f.i<?php echo $i?>subscriber_mname.value=f.form_mname.value;
-  f.i<?php echo $i?>subscriber_lname.value=f.form_lname.value;
-  f.i<?php echo $i?>subscriber_street.value=f.form_street.value;
-  f.i<?php echo $i?>subscriber_city.value=f.form_city.value;
-  f.form_i<?php echo $i?>subscriber_state.value=f.form_state.value;
-  f.i<?php echo $i?>subscriber_postal_code.value=f.form_postal_code.value;
-  if (f.form_country_code)
-    f.form_i<?php echo $i?>subscriber_country.value=f.form_country_code.value;
-  f.i<?php echo $i?>subscriber_phone.value=f.form_phone_home.value;
-  f.i<?php echo $i?>subscriber_DOB.value=f.form_DOB.value;
-  if(typeof f.form_ss!="undefined")
-    {
-        f.i<?php echo $i?>subscriber_ss.value=f.form_ss.value;  
-    }
-  f.form_i<?php echo $i?>subscriber_sex.value = f.form_sex.value;
-  f.i<?php echo $i?>subscriber_employer.value=f.form_em_name.value;
-  f.i<?php echo $i?>subscriber_employer_street.value=f.form_em_street.value;
-  f.i<?php echo $i?>subscriber_employer_city.value=f.form_em_city.value;
-  f.form_i<?php echo $i?>subscriber_employer_state.value=f.form_em_state.value;
-  f.i<?php echo $i?>subscriber_employer_postal_code.value=f.form_em_postal_code.value;
-  if (f.form_em_country)
-    f.form_i<?php echo $i?>subscriber_employer_country.value=f.form_em_country.value;
- }
-}
-
-<?php } ?>
-
-function popUp(URL) {
- day = new Date();
- id = day.getTime();
- top.restoreSession();
- eval("page" + id + " = window.open(URL, '" + id + "', 'toolbar=0,scrollbars=1,location=0,statusbar=0,menubar=0,resizable=1,width=400,height=300,left = 440,top = 362');");
-}
-
-function checkNum () {
- var re= new RegExp();
- re = /^\d*\.?\d*$/;
- str=document.demographics_form.monthly_income.value;
- if(re.exec(str))
- {
- }else{
-  alert("<?php xl('Please enter a monetary amount using only numbers and a decimal point.','e'); ?>");
- }
-}
-
-// Indicates which insurance slot is being updated.
-var insurance_index = 0;
-
-// The OnClick handler for searching/adding the insurance company.
-function ins_search(ins) {
-    insurance_index = ins;
-    return false;
-}
+  <head>
+    <?php html_header_show();?>
```

Comment:
html_header_show is an empty function

Reply:
Not actually in the scope of this PR though...

Reply:
@aethelwulffe jdew is working on this i think